firmware/esp32/client/BLEClient.h

```diff
@@ -0,0 +1,42 @@
+#ifdef USE_BLE
+#ifndef BLECLIENT_H
+#define BLECLIENT_H
+
+#include "../DisasterClient.h"
+#include "../DisasterServer.h"
+
+#include <BLEUtils.h>
+#include <BLEServer.h>
+#include <BLEDevice.h>
+#include <BLEAdvertising.h>
+#include <BLE2902.h>
+#include <esp_bt_device.h>
+
+#include "server/DisasterRadio.h"
+#include "middleware/HistoryReplay.h"
+
+extern DisasterRadio *radio;
+extern DisasterHistory *history;
+extern bool sdInitialized;
+extern bool spiffsInitialized;
+extern bool displayInitialized;
+extern bool loraInitialized;
+
+class BleDrClient : public DisasterClient
```

Comment:
Can we call this `BLEClient` for consistency with the others?

Reply:
sounds reasonable to me, believe it or not, I like consistency.